edge/test/integration/device/device_test.go

```diff
@@ -31,6 +32,38 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+//getting Devicestate from subscribed MQTT topic
+var Devicestate string
+
+//getting Device_state field from subscribed MQTT topic,Unmarshalling into structure variable
+var Device_state Device_Update
+
+type Device_Update struct {
```

Comment:
camel case please.
Applicable to all similar cases below

Reply:
ok